CMakeLists.txt

```diff
@@ -54,6 +72,12 @@ link_directories(
   ${OFED_PREFIX}/lib
   ${LEVELDB_PREFIX}/lib
 )
+if(${FREEBSD})
```

Comment:
@tchaikov 
This is an extract from `cmake --help-command if`:
```
The if command was written very early in CMake's history, predating
the ``${}`` variable evaluation syntax, and for convenience evaluates
variables named by its arguments as shown in the above signatures.
Note that normal variable evaluation with ``${}`` applies before the if
command even receives the arguments.  Therefore code like::

 set(var1 OFF)
 set(var2 "var1")
 if(${var2})

appears to the if command as::

 if(var1)

and is evaluated according to the ``if(<variable>)`` case documented
above.  The result is ``OFF`` which is false.  However, if we remove the
``${}`` from the example then the command sees::

 if(var2)

which is true because ``var2`` is defined to "var1" which is not a false
constant.
```

And because of that I think I would prefer to keep writting ${var}, just in case there is any medling with constants and variables.